code/__DEFINES/~effigy/techweb_nodes.dm

```diff
@@ -0,0 +1,5 @@
+// Custom techwebs
```

Comment:
path for this should be `code/__DEFINES/~effigy/research/techweb_nodes.dm` to mirror upstream